src/material-experimental/mdc-slider/module.ts

```diff
@@ -10,11 +10,12 @@ import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
 import {MatCommonModule} from '@angular/material-experimental/mdc-core';
 import {MatSlider} from './slider';
+import {MatSliderThumb} from './slider-thumb';
 
 @NgModule({
   imports: [MatCommonModule, CommonModule],
-  exports: [MatSlider, MatCommonModule],
-  declarations: [MatSlider],
+  exports: [MatSlider, MatSliderThumb, MatCommonModule],
```

Comment:
Should we be exporting `MatSliderThumb` or is it just an internal thing?

Reply:
It's internal. I still don't fully understand `NgModule`s so this is the result of me trying a bunch of stuff to get this to work